atomic_scanners/container-capabilities-scanner/scanner.py

```diff
@@ -105,20 +107,29 @@ def template_json_data():
 
     if out is not None:
         json_out["Scan Results"]["Container capabilities"] = out
+
+        if out == "":
+            json_out["Summary"] = "No additional capabilities found."
+        else:
+            json_out["Summary"] = \
+                "Container image has few additional capabilities."
```

Comment:
if it is having additional capabilities, we are printing the capability details after this line right?

Reply:
This line will be available in the email. For full details user will have to click on the link to the scan report.